src/components/headerMobile/homeHeaderMobile.css

```diff
@@ -0,0 +1,97 @@
+.header-mobile {
+  background: #f7f7f7;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+  z-index: 9997;
+  border-top: none;
+  padding: 15px 0px 0 10px;
```

Comment:
what do you think about add equal spacing throughout the container? like padding: 15px;

![image](https://user-images.githubusercontent.com/38358239/152176484-838cec02-7d1c-4993-8112-e9bbb5943064.png)


before:
![image](https://user-images.githubusercontent.com/38358239/152176748-df6ebdf7-bc27-45e8-ab49-e159b407fbec.png)

Reply:
I changed rai